fluss-test-utils/src/main/java/com/alibaba/fluss/testutils/common/CommonTestUtils.java

```diff
@@ -144,7 +144,10 @@ public static void retry(Duration timeout, Executable assertion) {
                 if (System.currentTimeMillis() - startTime >= maxWaitMs) {
                     throw t;
                 }
-                LOG.info("Attempt failed, sleeping for {} ms, and then retrying.", waitMs);
+                LOG.info(
+                        "Attempt failed, sleeping for {} ms, and then retrying. caused by: {}\n",
+                        waitMs,
+                        t.getMessage());
```

Comment:
Thanks for the contribution @liyubin117 .  However, the intermediate error message should be the same with the final error message? Does it provide any additional information?

Reply:
@wuchong Thanks for your attention. the feature provides additional error stack info.
the previous log:
```
3155 [ForkJoinPool-1-worker-1] INFO  com.alibaba.fluss.testutils.common.CommonTestUtils [] - Attempt failed, sleeping for 1 ms, and then retrying.
```
the current log:
```
10965 [ForkJoinPool-1-worker-1] INFO  com.alibaba.fluss.testutils.common.CommonTestUtils [] - Attempt failed, sleeping for 1 ms, and then retrying. caused by: 
Expecting actual:
  [0, 1, 2]
to contain exactly in any order:
  [0]
but the following elements were unexpected:
  [1, 2]
```

Reply:
thanks for your reminds, I have added aggregated errors, we can see all occured errors instead of the latest error.